config/nginx.conf.template

```diff
@@ -149,10 +149,34 @@ http {
   # Verify chain of trust of OCSP response using Root CA and Intermediate certs
   ssl_trusted_certificate /etc/letsencrypt/live/${TELESCOPE_HOST}/chain.pem;
 
-  # Requests to the (api|portainer).* domain should be proxied to Traefik
+  # Requests to the Portainer app
   server {
     listen 443 ssl http2;
-    server_name ${API_HOST} ${PORTAINER_HOST};
+    server_name ${PORTAINER_HOST};
+
+    location /api/websocket/  {
+      proxy_cache_bypass 1;
+      proxy_set_header Upgrade $http_upgrade;
+      proxy_set_header Connection "upgrade";
+      proxy_pass  http://portainer:9000/api/websocket;
+    }
+
+    location / {
+      proxy_cache_bypass 1;
+      proxy_set_header Connection "";
+      proxy_pass http://portainer:9000/;
+    }
+
+    # Let's Encrypt route
+    location /.well-known/acme-challenge/ {
+      root /var/www/certbot;
+    }
```

Comment:
I think we only need this in the `${TELESCOPE_HOST}` location. You can remove it here and in `${API_HOST}`

Reply:
Thank you, fixed.